### DISCUSSION TITLE
Where does APKTool store build APKs on Windows 10?

### DISCUSSION THREAD

**Discussion by WavyRancheros** [iBotPeaches/Apktool#2536] [Q&A] [answered]
I tried to use APKTool to modify a manifest in my apk, but I can't find the generated APK?

I looked into AppData/Local/apktool
I looked into the location where apktool.jar resides.
I also tried to give a full file path for the apk but nothing happens at that location.

I searched the entire filesystem for "*.apk" but I didn't find the one I just tried to generate.

Can anybody help?

**Comment by iBotPeaches** [comment, marked as answer]
If you don't use the `-o` parameter. It will default to `dist/{name}.apk`

So if are disassembling Twitter.apk, it will make a `Twitter` folder, inside at `Twitter/dist/Twitter.apk` the apk will reside. Also the `Twitter/build` folder has the files we need to build that dist.
